R/read_servers_helpers.R

```diff
@@ -291,49 +321,53 @@ get_id_column_name <- function(id_col_name, j, id_position) {
 #' \dontrun{
 #' result <- sql_select_entire_dataset(
 #'   table         = "author",
+#'   base_url      = "mysql-rfam-public.ebi.ac.uk",
+#'   user_name     = "rfamro",
+#'   password      = "",
 #'   dbms          = "MySQL",
 #'   driver_name   = "",
-#'   host          = "mysql-rfam-public.ebi.ac.uk",
 #'   database_name = "Rfam",
-#'   user          = "rfamro",
-#'   password      = "",
 #'   port          = 4497
 #' )
 #' }
 #' @keywords internal
 #' @noRd
-sql_select_entire_dataset <- function(table, dbms, driver_name, host,
-                                      database_name, user, password, port) {
+sql_select_entire_dataset <- function(table,
```

Comment:
Argument order seems to differ from the others. Can you explain why?

Reply:
The are changes in this function's signature. See [here](https://github.com/epiverse-trace/readepi/blob/rebased-arg-defaults/R/read_servers_helpers.R).

Maybe you need to `git pull` to get the recent version.

I have decided to pass arguments like `table` first to functions like `sql_select_entire_dataset()` because it is the main argument for this function. The remaining arguments that are needed to connect to the server come after them in the same order as defined in the `connect_to_server()` function. I made sure to preseve this order in all the functions.